robotoff/app/core.py

```diff
@@ -132,6 +138,39 @@ def get_insights(
     return query.iterator()
 
 
+def get_images(
+    with_predicted: Optional[bool] = False,
```

Comment:
I think we should name this parameter (here and in api) `with_predictions` instead of `with_predicted`.

The idea is to return images that have no prediction. (not images that have not been predicted…)